peer.go

```diff
@@ -2232,6 +2240,13 @@ func (p *peer) handleLinkFailure(failure linkFailureReport) {
 		}
 	}
 
+	// If this is a permanent failure, we will mark the channel borked.
+	if failure.linkErr.PermanentFailure && lnChan != nil {
```

Comment:
When would `lnChan` be nil?

Reply:
Just sanity check in case it is not in the map.